dts/arm/nxp/nxp_rw6xx_common.dtsi

```diff
@@ -16,6 +16,7 @@
 / {
 	chosen {
 		zephyr,entropy = &trng;
+		zephyr,nbu = &nbu;
```

Comment:
same comment as for mcxw about the alias, this is not what chosen node is for, and it's not a zephyr generic thing